src/ocp_postprocess.rs

```diff
@@ -218,6 +247,107 @@ pub(crate) async fn fix_olm_secret_hash_annotation(in_memory_etcd_client: &Arc<I
     Ok(())
 }
 
+fn verbosity(log_level: &str) -> String {
```

Comment:
Just to emphasize that this is not recert's own log verbosity

```suggestion
fn operator_log_verbosity(log_level: &str) -> String {
```